tests/functional/lib/fx-desktop.js

```diff
@@ -7,6 +7,10 @@ define([
   'intern/chai!assert'
 ], function (assert) {
 
+  function getRemote(context) {
+    return context.get ? context : context.remote;
```

Comment:
To tell you the truth, because I extracted it from other code, I didn't even think about that possibility.